iib/workers/tasks/opm_operations.py

```diff
@@ -907,3 +907,224 @@ def verify_operator_exists(
             is_package_in_index = True
             log.info("operator package %s found in index_db %s", operator_package, index_db_path)
     return is_package_in_index, index_db_path
+
+
+def set_opm_verison(index_image: str) -> str:
+    """
+    This function will verify if opm config is set in worker
+    if the config entry exists for the index image version
+    and return the opm version path to be used in the entire IIB request.
+    :param str index_image: from_index_image for the request
+    :return str opm_path: opm_path to be used for the entire request
+
+    """
+    global opm_path
```

Comment:
Maybe I missed that but I do not see this global variable defined anywhere. 

Somewhere at the top of this file should be `opm_path = "omp"`

Reply:
I added it earlier at the top of file as you have suggested, but I don't see any changes with adding that or in the current implementation, could you please share how declaring at the top as well can help here?